src/tactic/ui/table/statistic_wdg.py

```diff
@@ -518,13 +518,24 @@ def init_data(my):
 
         due_date = parser.parse(value)
 
+        import calendar
+        from datetime import datetime, timedelta
+
+        # convert the due date from UTC to local time
+        timestamp = calendar.timegm(due_date.timetuple())
+        local_dt = datetime.fromtimestamp(timestamp)
+        assert due_date.resolution >= timedelta(microseconds=1)
```

Comment:
what is this assert error for? In general , don't put assert error, tactic can't handle this type of error well...